paasta_tools/kubernetes_tools.py

```diff
@@ -553,6 +555,16 @@ def get_liveness_probe(
 
         return probe
 
+    def get_security_context(self) -> V1SecurityContext:
```

Comment:
since the function returns `None` in some cases, the return type is actually `Optional[V1SecurityContext]`.

Reply:
Ah, you're right. Let me fix it in the separated PR.